pkg/controller/errors.go

```diff
@@ -6,28 +6,36 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-type MultipleOwnerReferencesError error
+type MultipleOwnerReferencesError string
```

Comment:
Is this kind of a drive-by fix for this PR, or was it required to make things work? It doesn't seem related to the rest at first glance (our wide-spread bugs with error types notwithstanding).

Reply:
Yes, it was a drive-by, but to fix the bug where `shouldRetry` is always evaluated to `false` (see below).

https://github.com/bookingcom/shipper/blob/3015272fe0ec4b28bf67fdad0279d8e6f4e120ec/pkg/controller/installation/installation_controller.go#L187-L190

I'm fine to revert the related changes and create another PR for it.

Reply:
Ok cool, no need to split it. I'll grab this in the structured errors work either way.